src/postcss-loader.js

```diff
@@ -189,11 +189,15 @@ export default {
       };`
     }
     if (!shouldExtract && shouldInject) {
-      output += `\nimport styleInject from '${styleInjectPath}';\nstyleInject(css${
-        Object.keys(options.inject).length > 0 ?
-          `,${JSON.stringify(options.inject)}` :
-          ''
-      });`
+      if (typeof options.inject === 'function') {
+        output += options.inject(this.id)
```

Comment:
should we add more parameters in `inject` function?

example:
```ts
function inject(fileID: string, variableName: string, css: string): string;
```

need more discuss